src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1220,20 +1223,8 @@ class AddItemsPage extends Component {
     return Promise.resolve();
   }
 
-  submitRequest(lineItems) {
-    const nonEmptyLineItems = _.filter(lineItems, (val) => !_.isEmpty(val) && val.product);
-    this.saveRequisitionItems(nonEmptyLineItems)
-      .then(() => {
-        if (supports(this.state.values.origin?.supportedActivities, ActivityCode.APPROVE_REQUEST)) {
-          this.transitionToNextStep(RequisitionStatus.PENDING_APPROVAL);
-        } else {
-          this.transitionToNextStep(RequisitionStatus.REQUESTED);
-        }
-      });
-  }
```

Comment:
this isn't removed, just merged up into the saveAndTransitionToNextStep method. After talking with Kacper, we realized that my above TODO was correct and that flow was unreacheable so this is just a cleanup of that by removing the unused logic.